content/PGP/index.adoc

```diff
@@ -1,8 +1,11 @@
 == What is PGP? ==
 abbr:OpenPGP[Pretty Good Privacy] is an open standard for signing and encrypting. It enables RSA or ECC sign/encrypt operations using a private key stored on a smartcard (such as the YubiKey NEO), through common interfaces like PKCS#11.
 
-NOTE: PGP is not used for web authentication.
-
+[NOTE]
+===
```

Comment:
Sorry to be a pain, but it actually needs to be 4 equals symbols or it won't be rendered properly. Same for line 8.